source/install/desktop-msi-gpo.rst

```diff
@@ -120,3 +120,12 @@ Verify Group Policy settings in the installed Desktop app
 2. Verify the app loads the first server defined in the ``Edit group policy`` app.
 
    .. image:: ../images/desktop/msi_gpo/msi_gpo_installation_test_00018.png
+
+Silent Installation Guide
+--------------------------------------------
+
+Silent installation of MSI can be performed with the following command:
```

Comment:
Consider changing this passive sentence to be active (for consistency with the remainder of this document).  Suggestion: "Perform a silent installation of MSI by running the following command:"